nodejs/eks/userdata.ts

```diff
@@ -300,6 +310,13 @@ function createNodeadmUserData(
         });
     }
 
+    // add extra nodeadm options if provided
+    if (args.nodeadmExtraOptions) {
+        for (const option of args.nodeadmExtraOptions) {
+            parts.push(option);
```

Comment:
Will this override existing settings or how does the merging work?

Reply:
nodeadm will evaluate all `NodeConfig` sections and merge them together (deep merge) in the order they were specified.
So user settings overwrite base settings.